### PR TITLE
Update another 1.00 software version

### PR DESCRIPTION
**Pull request by delacor**

### CODE REVIEW

**Comment by simonebortolin** [comment]
please remove (?) if you dont know the year

File: _ont/ont-zyxel-pmg3000-d20b.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ The stick has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be a
 ## List of software versions
 - V1.00(ABVJ.0)b3s (2020)
 - V1.00(ABVJ.0)b3i (2020)
+- V1.00(ABVJ.0)b3v (?)
```
Comment (simonebortolin) [inline _ont/ont-zyxel-pmg3000-d20b.md:58]:
```suggestion
- V1.00(ABVJ.0)b3v
- ```

**Review by simonebortolin — CHANGES_REQUESTED** [review]

**Review by simonebortolin — APPROVED** [review]

File: _ont/ont-zyxel-pmg3000-d20b.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ The stick has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be a
 ## List of software versions
 - V1.00(ABVJ.0)b3s (2020)
 - V1.00(ABVJ.0)b3i (2020)
+- V1.00(ABVJ.0)b3v
+- ```
```
Comment (simonebortolin) [inline _ont/ont-zyxel-pmg3000-d20b.md]: ummh what

**Review by simonebortolin — APPROVED** [review]

File: _ont/ont-zyxel-pmg3000-d20b.md (hunk 2 above)
Comment (delacor) [reply _ont/ont-zyxel-pmg3000-d20b.md]: Yes, this is fixed, I also seen it in the preview :D

**Comment by simonebortolin** [comment]
sorry it didn't let me merge the PR until now.

**Pull request merged**